_posts/2023-08-23-cockpit-299.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Cockpit 299
+author: tmatus
+date: '2023-08-23'
+tags: cockpit
+slug: cockpit-299
+category: release
+summary: ''
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 299:
+
+
+## Kdump: Show location of kdump to verify the successful configuration test
+
+You can now see the location where kdump can be found if testing of kdump settings is successful.
+
+![Screenshot from 2023-08-23 11-17-45]({{ site.baseurl }}/images/299-kdump-show-location-of-kdump-to-verify-the-successful-configuration-test.png)
+
+## Storage: Support for no-overprovisioning with Stratis
+
+A Stratis pool can be put into a "no-overprovisioning" mode where the bad effects of running out
+of space can be avoided via careful management of filesystem sizes. Cockpit now supports this mode.
+
+## Storage: Cockpit can now add caches to encrypted Stratis pools
+
+Encrypted caches are a new feature in Stratis 3.5, and now you can use it also from Cockpit.
+
+
+## Try it out
+
+Cockpit 299 is available now:
+
+* [For your Linux system](https://cockpit-project.org/running.html)
+* [Cockpit Client](https://flathub.org/apps/details/org.cockpit_project.CockpitClient)
+
+* [Cockpit Source Tarball](https://github.com/cockpit-project/cockpit/releases/tag/299)
+* [Cockpit Fedora 40](https://bodhi.fedoraproject.org/updates/FEDORA-2023-4d17532ade)
+* [Cockpit Fedora 39](https://bodhi.fedoraproject.org/updates/FEDORA-2023-b19a60b701)
+* [Cockpit Fedora 38](https://bodhi.fedoraproject.org/updates/FEDORA-2023-bbb476ce91)
+* [Cockpit Fedora 37](https://bodhi.fedoraproject.org/updates/FEDORA-2023-1ab32d4881)
```

Comment:
We don't have to show F39/F40 these are not released yet.

Reply:
ok, removed